demos/JWST/S5_template.ecf

```diff
@@ -27,6 +27,10 @@ run_bound       'multi'
 run_sample      'unif'
 run_tol         0.1
 
+#GP inputs
```

Comment:
Should note that a GP can be used in run_myfuncs comment above

Reply:
Should also update the ecf templates in the docs/media folder and the eureka/testing/NIRCam and eureka/testing/NIRSpec folders